certified-connectors/Plumsail Forms/apiDefinition.swagger.json

```diff
@@ -19,72 +19,72 @@
   "produces": [],
   "paths": {
     "/api/attachments": {
-      "delete": {
+      "get": {
         "tags": [
           "Attachments"
         ],
-        "operationId": "DeleteAttachment",
-        "consumes": [
-          "application/json-patch+json",
-          "application/json",
-          "text/json",
-          "application/*+json"
+        "operationId": "DownloadAttachment",
+        "produces": [
+          "application/json"
         ],
         "parameters": [
           {
-            "in": "body",
+            "in": "query",
             "name": "fileUrl",
             "required": true,
-            "schema": {
-              "format": "uri",
-              "type": "string"
-            },
+            "type": "string",
+            "format": "uri",
             "x-ms-summary": "File URL",
             "description": "File URL"
           }
         ],
         "responses": {
-          "204": {
-            "description": "Success"
+          "200": {
+            "description": "The contents of the attachment",
+            "schema": {
+              "format": "binary",
+              "title": "Result file",
+              "description": "The contents of the attachment",
+              "type": "string"
+            }
           }
         },
         "x-ms-visibility": "important",
-        "summary": "Delete attachment",
-        "description": "Deletes an attachment by its URL"
+        "summary": "Download attachment",
+        "description": "Downloads an attachment by its URL"
       },
-      "post": {
+      "delete": {
         "tags": [
           "Attachments"
         ],
-        "operationId": "DownloadAttachmentPost",
```

Comment:
Hello @jusper-dk, This operation has been deleted. This change breaks existing flows. May I ask you please to confirm that no existing users will be affect with this change?
Thank you so much.

Reply:
Yes, I confirm that this operation has never been published.